yarn.lock

```diff
@@ -24,7 +24,7 @@
   dependencies:
```

Comment:
this file is also not needed

Reply:
I don't think I added this file as part of the PR.
I thought that the root-level yarn.lock is used by lerna/yarn workspaces. (as opposed to having a yarn.lock in every package)

Reply:
oh yeah, it was werid, it shown that lock inside package, nvm this comment